services/mempool/replyfee.go

```diff
@@ -7,10 +7,10 @@ package mempool
 
 import "github.com/Qitmeer/qng/core/types"
 
-// calcMinRequiredTxRelayFee returns the minimum transaction fee required for a
+// CalcMinRequiredTxRelayFee returns the minimum transaction fee required for a
 // transaction with the passed serialized size to be accepted into the memory
 // pool and relayed.
-func calcMinRequiredTxRelayFee(serializedSize int64, minRelayTxFee types.Amount) int64 {
+func CalcMinRequiredTxRelayFee(serializedSize int64, minRelayTxFee types.Amount) int64 {
```

Comment:
its better to refactor the original function (instead of change function name) to call a  util.CalcFee function. then all `mempool.go` changes can be eliminated and your new code need not to introduce `mempool` package.